README.md

```diff
@@ -241,3 +241,83 @@ We're going to try spinning this out into its own library; see: https://github.c
 	    hash: true,             // by default this is FALSE
          }).addTo(map);
 	
+## Add a new layer
```

Comment:
Lets rename this as Steps to add a new layer.
Also lets move these steps into a seperate steps.md file. Awesome :)